CHANGELOG.md

```diff
@@ -118,6 +119,7 @@
     Bug #4945: Poor random magic magnitude distribution
     Bug #4947: Player character doesn't use lip animation
     Bug #4948: Footstep sounds while levitating on ground level
+    Bug #4949: Incorrect particle lighting when force shaders = true
```

Comment:
Again, these changes won't be a part of 0.46.0. Put them to 0.47.0 milestone.

Reply:
haven't understood...